guides/contributor/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenTelemetry New Contributor Guide
+
+Welcome to OpenTelemetry! We're excited for you to join us. This guide will help
+you get started by providing an overview of the project, how we work together,
+and point you towards helpful resources.
+
+## Table of Contents
+
+- [What is OpenTelemetry?](#what-is-opentelemetry)
+- [Prerequisites](#prerequisites)
+
+## What is OpenTelemetry?
+
+OpenTelemetry is a framework for application telemetry data. If you've ever
+checked how much memory a process is using on your computer, or looked at the
+size of a file, you've already interacted with telemetry! Cloud-native systems
+create, emit, and process many millions of telemetry points every second as they
+run. In addition, the developers of cloud-native software need to describe what
```

Comment:
maybe remove "cloud-native"? though we are under the CNCF, so maybe not?

Reply:
and if we keep it, [remove the hyphen](https://www.cncf.io/blog/2018/09/04/the-cloud-native-computing-foundation-cncf-style-guide/)

```suggestion
size of a file, you've already interacted with telemetry! Cloud-native systems
create, emit, and process many millions of telemetry points every second as they
run. In addition, the developers of cloud native software need to describe what
```